src/logic/calculate.js

```diff
@@ -0,0 +1,98 @@
+import operate from './operate';
+
+const isNumber = item => !!item.match(/[0-9]+/);
+
+export default function calculate(data, buttonName) {
```

Comment:
According to the instructions on the pathwright, we need to pass an object as an argument.
_Screenshot from [pathwright](https://microverse.pathwright.com/library/react-redux-v11/184438/path/step/106376018/
);_
**![image](https://user-images.githubusercontent.com/31897434/111031643-db5f6900-8408-11eb-8e94-e3e786d10bdd.png)**


The argument passed to the `function calculate` should be an object, we can achieve this by using destructuring.
```
function calculate(data, buttonName) {
  let { total, next, operation } = data;
.....
}
```